Lib/fontTools/misc/plistlib.py

```diff
@@ -5,18 +5,8 @@
 from base64 import b64encode, b64decode
 from numbers import Integral
 
-try:
-    from collections.abc import Mapping # python >= 3.3
-except ImportError:
-    from collections import Mapping
-
-try:
-    from functools import singledispatch
-except ImportError:
-    try:
-        from singledispatch import singledispatch
-    except ImportError:
-        singledispatch = None
+from collections.abc import Mapping
+from functools import singledispatch
```

Comment:
In plislitb module if you look further down there is an if singldispatch is not None branch, you can simplify that one since we no longer support the case of singldispatch not being present.

Also there is a different PY2 vs PY3 treatment for use_builtin_types argument default value, that can be simplified too

Reply:
Should I make `use_builtin_types` a no-op and print a deprecation warning when it's used?